sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/CreateNewSampleSteps.java

```diff
@@ -722,6 +679,59 @@ public CreateNewSampleSteps(
           softly.assertAll();
         });
 
+    And(
+        "I select Sent dispatched checkbox",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(SAMPLE_SHIPPED_CHECKBOX);
+          webDriverHelpers.clickOnWebElementBySelector(SAMPLE_SHIPPED_CHECKBOX);
+        });
+
+    And(
+        "I select Received checkbox",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(SAMPLE_RECEIVED_CHECKBOX);
+          webDriverHelpers.clickOnWebElementBySelector(SAMPLE_RECEIVED_CHECKBOX);
+        });
+
+    Then(
+        "I check is Sent dispatched Date and Received Date fields required",
+        () -> {
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(DATE_SAMPLE_SEND);
+
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(DATE_SAMPLE_RECEIVED);
+        });
+
+    And(
+        "I click Add Pathogen test in Sample creation page in German",
+        () -> webDriverHelpers.clickOnWebElementBySelector(ADD_PATHOGEN_TEST_BUTTON_DE));
+
+    And(
+        "I check DATE AND TIME OF RESULT field",
+        () -> {
+          TimeUnit.SECONDS.sleep(10);
+          webDriverHelpers.getWebElement(DATE_AND_TIME_OF_RESULTS_DE);
+        });
+
+    And(
```

Comment:
please check if this method doesn't already exist

Reply:
All methods with SAVE_SAMPLE_BUTTON in CreateNewSampleSteps contains some other actions. There is no simple "click on save button" method.